x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/action/DeleteDataFrameAnalyticsAction.java

```diff
@@ -40,7 +40,7 @@ private DeleteDataFrameAnalyticsAction() {
         public Request(StreamInput in) throws IOException {
             super(in);
             id = in.readString();
-            if (in.getVersion().onOrAfter(Version.CURRENT)) {
+            if (in.getVersion().onOrAfter(Version.V_8_0_0)) {
```

Comment:
Since https://github.com/elastic/elasticsearch/pull/50589 is merged, shouldn't the version be `V_7_6_0`?

Reply:
since BWC has passed with the version being wrong, I am guessing we don't have a BWC test that deletes analytics jobs do we?

Reply:
Ah, 🤦‍♂ 

Indeed. I'll add one that creates and deletes a job in the mixed cluster.